catroid/src/main/java/org/catrobat/catroid/ui/BaseActivity.java

```diff
@@ -137,4 +147,25 @@ public void addToRequiresPermissionTaskList(RequiresPermissionTask task) {
 	public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
 		permissionRequestActivityExtension.onRequestPermissionsResult(this, requestCode, permissions, grantResults);
 	}
+
+	public static void removeExifFromPNG(File file) {
+		try {
+			FileInputStream fileInputStream = new FileInputStream(file.getAbsolutePath());
+			BufferedInputStream bufferedInputStream = new BufferedInputStream(fileInputStream);
+			byte[] bMapArray = new byte[bufferedInputStream.available()];
+			bufferedInputStream.read(bMapArray);
+			Bitmap bitmap = BitmapFactory.decodeByteArray(bMapArray, 0, bMapArray.length);
+
+			ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+			bitmap.compress(Bitmap.CompressFormat.JPEG, 100, byteArrayOutputStream);
```

Comment:
Unfortunately, compressing all images to the `JPEG` format is not a good idea. For example if the chosen image has `png` format we loose all the transparency information. You can easily try this out with the "normal" actors from the Media library as they all have transparent background.
Moreover if I use image file formats, which are not supported by Catroid, e.g. `.svg` instead of showing an error message the app crashes. For reproduction see the attached screencast (the file I selected the second time has `.svg` format)

https://user-images.githubusercontent.com/60399652/111780212-603dfd00-88b7-11eb-8c57-0b9a7b376e91.mp4